NoteFind/Models/Note.h

```diff
@@ -12,19 +12,22 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface Note : PFObject <PFSubclassing>
 
-@property (strong, nonatomic) NSString *noteID;
+
 @property (strong, nonatomic) NSString *userID;
 
 @property (strong, nonatomic) PFFileObject *note;
 @property (strong, nonatomic) PFUser *author;
-@property (strong, nonatomic) NSString *caption;
+@property (strong, nonatomic) NSString *noteDescription;
 
 @property (strong, nonatomic) NSArray *tags;
-@property (strong, nonatomic) NSString *textbook;
+@property (strong, nonatomic) NSArray *textbooks;
+@property (strong, nonatomic) NSString *title;
```

Comment:
Change all NSString property types to copy

Reply:
resolved